public/app/features/explore/spec/queryLibrary.test.tsx

```diff
@@ -105,7 +105,6 @@ describe('QueryLibrary', () => {
     await waitForExplore();
     await openQueryLibrary();
     await assertQueryLibraryTemplateExists('loki', 'Loki Query Template');
-    await assertQueryLibraryTemplateExists('elastic', 'Elastic Query Template');
```

Comment:
Since we are setting active data source in Explore as the default filter, Query Library will only show items for that active filter